### PR TITLE
EDGE-607: Removed from and digits from verify request schema

### PR DESCRIPTION
**Pull request by psherk-bw**
### What
Remove unused from and digits fields

See related PR's
- https://github.com/Bandwidth/mfa_api/pull/69
- https://github.com/Bandwidth/node-mfa/pull/7

### Why
Unused for verification

### CODE REVIEW

**Comment by psherk-bw** [comment]
Need to wait until https://github.com/Bandwidth/node-mfa/pull/7 is merged with updated sdk version, and then update our `package.json`

File: controller/authenticate/twofactor.js [hunk 1]
```diff
@@ -26,12 +26,10 @@ const sendSmsCode = async (phoneNumber) => {
 
     return await controller.createMessagingTwoFactor(exampleAppConfig.accountId,
       new twofactor.TwoFactorCodeRequestSchema({
-        from: exampleAppConfig.phoneNumber,
         to: phoneNumber,
         applicationId: exampleAppConfig.applicationId,
         scope: 'authorization',
-        message: 'Your temporary {NAME} {SCOPE} code is {CODE}',
-        digits: 6
+        message: 'Your temporary {NAME} {SCOPE} code is {CODE}'
```
Comment (carrington-bw) [inline controller/authenticate/twofactor.js]: I think you want to change the verification request below

File: controller/authenticate/twofactor.js [hunk 2]
```diff
@@ -26,12 +26,10 @@ const sendSmsCode = async (phoneNumber) => {
```
Comment (cejohnson-bw) [inline controller/authenticate/twofactor.js]: While you're in here, it looks like the documentation for these methods has drifted a bit if you don't mind either updating or perhaps removing the example objects.

File: controller/authenticate/twofactor.js (hunk 1 above)
Comment (psherk-bw) [reply controller/authenticate/twofactor.js]: thank you! updated

**Review by carrington-bw — APPROVED** [review]

File: controller/authenticate/twofactor.js (hunk 2 above)
Comment (psherk-bw) [reply controller/authenticate/twofactor.js]: updated

**Review by cejohnson-bw — APPROVED** [review]
LGTM

**Pull request merged**